app/pages/multisig/updates/CreatePLT/CreatePltParametersView.tsx

```diff
@@ -0,0 +1,102 @@
+                <div className="body3 mono mB20">
+                    {initializationParameters.name}
+                </div>
+            </div>
+            <div>
+                <Label className="mB5">{fieldDisplays.moduleRef}:</Label>
+                <div className="body3 mono mB20">{moduleRef.toString()}</div>
+            </div>
+            <div>
+                <Label className="mB5">{fieldDisplays.metadataUrl}:</Label>
+                <div className="body3 mono mB20">
+                    {initializationParameters.metadata.url}
+                </div>
+            </div>
+            {initializationParameters.metadata.checksumSha256 && (
+                <div>
+                    <Label className="mB5">{fieldDisplays.metadataHash}:</Label>
+                    <div className="body3 mono mB20">
+                        {uint8ArrayToHex(
+                            initializationParameters.metadata.checksumSha256
+                                ?.buffer
+                        )}
+                    </div>
+                </div>
+            )}
+            <div>
+                <Label className="mB5">
+                    {fieldDisplays.governanceAccount}:
+                </Label>
+                <div className="body3 mono mB20">
+                    {initializationParameters.governanceAccount.address.address}
+                </div>
+            </div>
+            <div>
+                <Label className="mB5">{fieldDisplays.decimals}:</Label>
+                <div className="body3 mono mB20">{decimals.toString()}</div>
+            </div>
+            <div>
+                <Label className="mB5">{fieldDisplays.initialSupply}:</Label>
+                <div className="body3 mono mB20">
+                    {initializationParameters.initialSupply?.value?.toString() ??
+                        '0'}
+                </div>
+            </div>
+            <div>
+                <Label className="mB5">{fieldDisplays.allowList}:</Label>
+                <div className="body3 mono mB20">
+                    {initializationParameters.allowList?.toString() ?? 'False'}
+                </div>
+            </div>
+            <div>
+                <Label className="mB5">{fieldDisplays.denyList}:</Label>
+                <div className="body3 mono mB20">
+                    {initializationParameters.denyList?.toString() ?? 'False'}
+                </div>
+            </div>
+            <div>
+                <Label className="mB5">{fieldDisplays.mintable}:</Label>
+                <div className="body3 mono mB20">
+                    {initializationParameters.mintable?.toString() ?? 'False'}
+                </div>
+            </div>
+            <div>
+                <Label className="mB5">{fieldDisplays.burnable}:</Label>
+                <div className="body3 mono mB20">
+                    {initializationParameters.burnable?.toString() ?? 'False'}
+                </div>
+            </div>
```

Comment:
There's so much data to represent here, that the view of the proposal does not look very nice in the proposal overview. I don't know if it warrants changing it.

If you do want to change it, you'd need to update it in `UpdateInstructionDetails.module.scss`. I found that `25px` between them looks nice 🤷